src/lib/viewers/controls/color-picker/ColorPickerControl.tsx

```diff
@@ -44,7 +44,13 @@ export default function ColorPickerControl({
                 type="button"
                 {...rest}
             >
-                <div className="bp-ColorPickerControl-swatch" style={{ backgroundColor: activeColor }} />
+                <div
+                    className={classNames('bp-ColorPickerControl-highlight', {
+                        'bp-has-attention': isColorPickerToggled,
```

Comment:
The naming here is a little confusing, as we have a hook called `useAttention` that has different behavior. Can we name this `bp-is-active`? Should this class be on the `bp-ColorPickerControl-button` element, as well, since that is what actually is toggled?

Reply:
Yes that makes sense